UnityAds/src/main/java/com/mopub/mobileads/UnityBanner.java

```diff
@@ -80,50 +87,29 @@ protected void loadBanner(Context context, CustomEventBannerListener customEvent
         }
     }
 
-    private void initNoRefreshMetaData(Context context) {
-        MetaData metaData = new MetaData(context);
-        metaData.set("banner.refresh", false);
-        metaData.commit();
-    }
-
     @Override
     protected void onInvalidate() {
-        UnityRouter.getBannerRouter().removeListener(placementId);
-        UnityRouter.getInterstitialRouter().removeListener(placementId);
-        UnityBanners.destroy();
-
-        bannerView = null;
+        if (mBannerView != null) {
+            mBannerView.destroy();
+        }
+        mBannerView = null;
         customEventBannerListener = null;
     }
 
     @Override
-    public void onUnityBannerLoaded(String placementId, View view) {
+    public void onBannerLoaded(BannerView bannerView) {
         MoPubLog.log(LOAD_SUCCESS, ADAPTER_NAME);
         MoPubLog.log(SHOW_ATTEMPTED, ADAPTER_NAME);
         MoPubLog.log(SHOW_SUCCESS, ADAPTER_NAME);
 
         if (customEventBannerListener != null) {
-            customEventBannerListener.onBannerLoaded(view);
-            this.bannerView = view;
+            customEventBannerListener.onBannerLoaded(bannerView);
+            mBannerView = bannerView;
```

Comment:
We should not need to set the mBannerView in this method as it is created and set above.